frontend/components/header/index.tsx

```
@@ -1,43 +1,51 @@
 import { getAuthorizedUserByEmail } from "@/lib/requests/authorized-user";
 import { DarkMode } from "../explore/dark-mode";
 import { Logo } from "./logo";
+import { useState } from "react";
 
-export async function Header() {
-  const user = await getCurrentUser();
+export function Header({
+  user,
+  authorizedUser,
+}: {
+  user: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)